README.md

```diff
@@ -388,6 +402,8 @@ Yapmış olduğumuz değişikliklerden sonra  aşağıdaki adımları izleyerek
 - https://learnxinyminutes.com/docs/markdown/
 - https://www.ibm.com/docs/en/SSYKAV?topic=train-how-do-use-markdown
 
+
+
 ### Contributors
 
 - Ufuk Yetişkin
```

Comment:
Buraya da yazılan dokümanı linklersen tamamdır

Reply:
yanlış anlaşılmışım Contributors -> - Ufuk Yetişkin'i linklemeni istemiştim.

Bir de branch'ini rebase origin/master yapmıyorsun o yüzden branch'in eski kalıyor ve karışıklıklara sebep oluyor. PR açmadan önce her seferinde rebase yaparsan daha temiz olur.

Örneğin dokümanın(Readme.md) son halinde bu bölüm yok ama senin branch'in eski kaldığı için hala bu bölümü barındırıyor.
![image](https://user-images.githubusercontent.com/36634192/158530972-e52ec795-4ace-4ab1-93b9-198f75837b40.png)